inlang/source-code/sdk2/src/bundle-id/bundle-id.ts

```diff
@@ -11,20 +11,20 @@ export function generateBundleId() {
 }
 
 // TODO needed for what?
-// export function humanIdHash(value: string, offset: number = 0) {
-// 	// Seed value can be any arbitrary value
-// 	const seed = 42
+export function humanIdHash(value: string, offset: number = 0) {
```

Comment:
Rename to `stableBundleId` to increase the understanding of what the function is for?

```suggestion
export function humanIdHash(value: string, offset: number = 0) {
export function stableBundleId(value: string, offset: number = 0){
```